test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterTests.cs

```diff
@@ -34,8 +36,8 @@ public class PrometheusExporterTests
     {
         private readonly ITestOutputHelper output;
         private const int metricPushIntervalMsec = 100;
-        private const int waitDuration = metricPushIntervalMsec + 100;
-
+        private const int waitDuration = metricPushIntervalMsec + 10;
```

Comment:
Since there is no repetition on the output anymore the wait on the test becomes critical for the success of the tests, likely we need a better way to synchronize it. I ran it many times successfully on my box but CI may be more different, if this becomes noisy we need to change it.